.github/workflows/lms.yml

```
@@ -0,0 +1,52 @@
+name: lms-rust
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/lms.yml"
+      - "lms/**"
+      - "Cargo.*"
+  push:
+    branches: master
+
+defaults:
+  run:
+    working-directory: .
```

I would expect this to be:

```suggestion
    working-directory: lms
```